lib/senders/rust_mailer.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request')
+
+function sendMail(emailConfig, cb) {
+    const options = {
+        url: 'http://127.0.0.1:8001/send',
```

Comment:
Oh yeah, let's also make sure the host and port that we use here also come from config.